server/views/pages/reviewsTable.njk

```diff
@@ -107,4 +132,27 @@
 {% block bodyEnd %}
   {{ super() }}
   <script src="/assets/gaSendEvents.js"></script>
+
+  <script type="text/javascript" nonce="{{ cspNonce }}">
+    function hideBanner(event) {
+      const dismissLink = $(this)
+      const id = dismissLink.data('notification-id')
+
+      const notificationBanner = dismissLink.parents('.govuk-notification-banner')
+      const csrf = $('#_csrf').val()
+
+      $.ajax({
+        type: 'POST',
+        url: '/notification/dismiss',
+        headers: {
+          'X-CSRF-Token': csrf,
+        },
+        data: { id: id },
+      })
+        .done(() => notificationBanner.hide())
+        .fail(() => window.location = '/')
```

Comment:
if the notification fails to dismiss, i figure the user might as well stay where they are?

Reply:
I wanted to ask for your opinion about this actually.

We could do nothing (on error notification stays), hide anyway (on reload banner would reappear which could be confusing)

Reply:
Cool, I've removed the redirect: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/362/commits/c4d978739026b54da598bb830ed1bf09da5391e6